Frontend/src/views/WalksOverview.vue

```diff
@@ -1,12 +1,81 @@
 <template>
-  <!--vuetify div-->
   <v-container>
-    <p>WALKS OVERVIEW</p>
+    <p>Chose a walk</p>
+    <v-row>
+      <v-col cols="4">
+        <v-card @click="goToDetail(1)">
+          <v-img v-if="hasDonePhotowalk(1)"
+            :src="`/images/map_stadtpark_Done.png`"
+            :lazy-src="`images/gradient.png`"
+            class = "rounded-lg">
+          </v-img>
+          <v-img v-else
+             :src="`/images/map_stadtpark_notDone_small.png`"
+             :lazy-src="`images/gradient.png`"
+             class = "rounded-lg">
+          </v-img>
```

Comment:
Doppelter Code. Geht das nicht in einem v-img mit nem if else Einzeiler im src Parameter?

Reply:
Ergibt Sinn. Würde ich erstmal so lassen und dann zusammen mit dem noch fehlenden hover und styling anpassen.

Reply:
Ok